### ISSUE TITLE
Less precise ticks in scoreboard

### ISSUE THREAD

**Issue by stefano-maggiolo** [cms-dev/cms#648]
Since we don't make an effort to implement all the rules about scoring and medals, it would be better to avoid giving the impression of preciseness with the ticks at the 1/12, 1/4 and 1/2 on the right part of the graph in RWS.

My proposal is [shown here](https://drive.google.com/open?id=0B-aKUcKSQbEAaHlPUUZoWDdkWU0). Feel free to propose some other way.

**Comment by smadbe** [comment]
The debate at the last IOI was more because they considered the ranking as final while we know that the ranking can still change a lot (disqualification, appeals, ...). I don't think having a blurrier limit would have resulted in a different debate. Couldn't we just have a mention under the graph (bottom left) saying that both the ranking and the medal cuts are just indicative, not official and might change in the future ?

**Comment by stefano-maggiolo** [comment]
Yes, see https://github.com/cms-dev/cms/issues/649 :)

Anyhow, I think it's better to have margin for errors since we don't implement all the rules, in particular what happens with contestants with the same scores. This includes when results are final, btw.

**Comment by lerks** [comment]
What about changing their positions? We could make a tick every quintile (i.e. at 20%, 40%, 60%, ...)? That way no tick would be at a medal cutoff point, but they would still help understand where someone is located...

**Comment by smadbe** [comment]
At the end, what people mainly wants to know when they follow the ranking live are the medal cuts, I don't think it is a good idea to remove them.

**Comment by lerks** [comment]
Yes, I agree, and that's why when I first wrote that overview in 2012 I put those "hints". I never intended them to be precise or official (because I didn't have the time to implement it), but apparently that wasn't made clear enough. I can see why the IOI community wants this to be changed.

I see three options: either we remove them completely (and possibly replace them with something like what I proposed in my previous comment), or we blur them or make it evident in some other way that they are approximate (as @stefano-maggiolo originally suggested), or we make them actually accurate, implementing the IOI rules about ties (that would probably also require #650 and a disclaimer that results may change with reevaluations/appeals). I consider the last option to be the most complicated and "risky" one. As for the other two I don't have strong preferences.

**Comment by smadbe** [comment]
Stefano's proposal is probably the safest then. But no need for an unclear large blur, otherwise people will reimplement their own proxy to have the actual ranking.
